docker/config.yaml

```diff
@@ -0,0 +1,17 @@
+Global:
+  ManageImages: true
+#  BootMountPoint: /boot/efi
+  DracutConfDir: /etc/zfsbootmenu/dracut.conf.d
+Components:
+  ImageDir: /boot/efi/EFI/void
+  Versions: 3
+  Enabled: false
+  syslinux:
+    Config: /boot/syslinux/syslinux.cfg
+    Enabled: false
+EFI:
+  ImageDir: /boot/efi/EFI/void
+  Versions: false
+  Enabled: true
+Kernel:
+  CommandLine: ro quiet loglevel=0
```

Comment:
```yaml
CommandLine: zfsbootmenu ro loglevel=4 nomodeset
```
Use `loglevel=4` to provide some useful warnings if need be, and `nomodeset` to avoid initializing the GPU.